wordpress-mobile/WordPress-iOS
# [Update Username] Update CoreData model and related models

Refs. #9705

This PR updates the CoreData model (version **88**), adding the property `usernameCanBeChanged` to the `AccountSettings` entity and to the `ManagedAccountSettings` managed object model. It also updates the map and inverse map between the `AccountSettings` and `ManagedAccountSettings` models.

I will update the Podfile as soon [#172](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/172) will be merged.

## To test:
- Run the app on a previous version to check the CoreData transition works as expected
- Tap _Me_ -> _Account Settings_ and check if everything works as before

Update release notes:
